plugins/http_plugin/tests/unit_tests.cpp

```diff
@@ -642,13 +644,14 @@ BOOST_FIXTURE_TEST_CASE(bytes_in_flight, http_plugin_test_fixture) {
 
    //load up some more requests that exceed max
    send_4mb_requests(32u);
-   //make sure got to the point http threads had responses queued
-   std::this_thread::sleep_for(std::chrono::seconds(1));
    //now rip these connections out before the responses are completely sent
    connections.clear();
    //send some requests that should work still
    send_4mb_requests(8u);
    r = drain_http_replies();
+   for (const auto& e : r) {
+      ilog( "response: ${r}, count: ${c}", ("r", std::string(obsolete_reason(e.first)))("c", e.second));
```

Comment:
I almost thought the `r` in `${r}` was the `r` in `r = drain_http_replies()`.

Reply:
Changed